### DISCUSSION TITLE
Add command to verify if it is needed to install dependencies to build a program

### DISCUSSION THREAD

**Discussion by radare** [vlang/v#11643] [Ideas]
**V version:** V 0.1.24 436603a
**OS:** macOS 10.15

**What did you do?**

build a program or chk if the deps are installed 

**What did you expect to see?**

$ v check .

return 0 or 1 depending if we need to do `v install .`
 
**What did you see instead?**

network requests and more than 1s of wait to finish the `v install ..` line

**Comment by spytheman** [comment]
There is no `v check .`

**Comment by radare** [comment]
i know, that's why im asking for it because i owuld find it interesting to have
